android/src/main/java/com/tailscale/ipn/ui/notifier/Notifier.kt

```diff
@@ -83,6 +84,10 @@ object Notifier {
       state.collect { currstate ->
         readyToPrepareVPN.set(currstate > Ipn.State.Stopped)
         tileReady.set(currstate >= Ipn.State.Stopped)
+
+        if (currstate >= Ipn.State.Starting) {
+          Libtailscale.maybeUpdateTUN()
```

Comment:
Can we add a comment to explain why we're calling into this?

Reply:
It feels to me like the UI shouldn't even do this.  If we always need to call this method when the state changes to !Starting, it should be done in goland and explained there.

Reply:
makes sense! moved this in to go.